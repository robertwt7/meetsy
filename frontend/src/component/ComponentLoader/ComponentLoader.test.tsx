import "@testing-library/jest-dom";

jest.mock("../ComponentLoader");