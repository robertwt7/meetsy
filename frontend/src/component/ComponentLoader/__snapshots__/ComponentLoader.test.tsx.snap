// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentLoader should match snapshot 1`] = `
<body>
  <div>
    <div
      class="flex h-full w-full items-center"
    >
      <div
        class="w-full text-center"
      >
        <span
          class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
          role="progressbar"
          style="width: 10%; height: 10%;"
        >
          <svg
            class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
            viewBox="22 22 44 44"
          >
            <circle
              class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
              cx="44"
              cy="44"
              fill="none"
              r="20.2"
              stroke-width="3.6"
            />
          </svg>
        </span>
      </div>
    </div>
  </div>
</body>
`;
